docs/Contribute/ContributeWrite/README.MD

```diff
@@ -4,7 +4,7 @@ By giving essential and concise informations on software, or creating these docu
 
 If you have writing skills, if you have a fair knowledge of the English language, then you can contribute to BTCPay Server or review the work of other contributors.
 
-Writing contributors can help in a number of places like the Software stack, the documentation or the directory for example.
+Writing contributors can help in a number of places like the software, the documentation or the directory for example.
```

Comment:
link to relevant sections may be useful

Reply:
```suggestion
Writing contributors can help in a number of places like the software, the documentation or the blog for example.
```

Forgot to delete directory and replace blog, since no longer relevant.